Upgrade to colored 2.0. The upgrade is trivial and lets us avoid two
copies of colored.
https://github.com/borntyping/rust-simple_logger/pull/49

Note that this patch applies to the normalized copy of Cargo.toml that lives
inside the crate, which is different to Cargo.toml committed to upstream git.

--- a/Cargo.toml        2022-01-10 15:04:29.005102916 +1100
+++ b/Cargo.toml        2022-01-10 15:06:32.589507547 +1100
@@ -31,7 +31,7 @@
 optional = true

 [dependencies.colored]
-version = "^1.6"
+version = "2"
 optional = true

 [dependencies.log]
